Uanion/Uanion.Application/Features/User/Queries/GetUsersList/UserListViewModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Uanion.Application.Features.User.Queries.GetUsersList;
```

Comment:
Need to add empty line between namespace and class.

Reply:
Done